resources/test/docker-compose-az-flexible-postgres-tmpl.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.8"
+services:
+  postgres:
+    image: ${IMAGE}
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_PASSWORD: "test1234"
+      POSTGRES_USER: "az_admin"
+    volumes:
+      - ./resources/test/az_flexible_postgres_init.sh:/docker-entrypoint-initdb.d/az_init.sh
+      - data:/usr/share/postgresql
+  extension-mark:
```

Comment:
why is this needed? It's simple to understand when facing an issue or google it? Maybe a comment as a hint?

Reply:
I've added comments.